### PR TITLE
DOC-10320 Add LiteCore Changes to Couchbase-lite 3.0.2 release notes

### PR DESCRIPTION
**Pull request by RayOffiah**
Added missing entries.

### CODE REVIEW

File: modules/android/pages/_partials/pn-issues-list.adoc [hunk 1]
```diff
@@ -6,12 +6,21 @@
 
 // tag::Fixed-3-0-2[]
 
+
 * https://issues.couchbase.com/browse/CBL-3301[CBL-3301] -- https://issues.couchbase.com/browse/CBL-3301[FullTextIndex.setLanguage() should accept null parameter]
 
 * https://issues.couchbase.com/browse/CBL-3299[CBL-3299] -- https://issues.couchbase.com/browse/CBL-3299[Fix NPE in finalizer]
 
+* https://issues.couchbase.com/browse/CBL-3224[CBL-3224] -- https://issues.couchbase.com/browse/CBL-3224[Call to `c4socket_closed` causes native crash]
```
Comment (osfameron) [inline modules/android/pages/_partials/pn-issues-list.adoc:14]: I hadn't really registered that you have to write the JIRA issue number *3 times* for these 😱 
(I should get my `jira:` macro merged in at some point maybe...)

**Review by osfameron — APPROVED** [review]
Looks good! 👍

File: modules/android/pages/_partials/pn-issues-list.adoc (hunk 1 above)
Comment (RayOffiah) [reply modules/android/pages/_partials/pn-issues-list.adoc:14]: Yes, some of the changes apply to more than one component. 
The LiteCore ones might apply to all of them (look at the list of comments to see which components are affected by a rebuild of the LiteCore module).

Comment (osfameron) [reply modules/android/pages/_partials/pn-issues-list.adoc:14]: Ah, separate question, though that's really useful - I think perhaps next time there's a release, I could do it with your guidance? As it feels a little abstract still to me.

But I just meant that for each line, you're having to write e.g. `CBL-3224` 3 times (twice for the ticket number, and once again for the description) which seems like a pain and error prone.
(Just a drive-by 🤔 really, and reminder to self that I wrote that jira: macro and never integrated it)

Comment (RayOffiah) [reply modules/android/pages/_partials/pn-issues-list.adoc:14]: Oh, I see! 

Yup, that's quite painful. I just followed the existing pattern, though I'm not really sure why it needs to be two URLs when a single one with some text would have done the trick.

So if you resurrect your JIRA macro that would save a lot of typing.

Yup, I can certainly walk you through it if you need to do one.  

I can explain how it works, but I can't really explain why it was done like this. My best guess that this was part of some automation plan that was going to generate each of the component's issue list files as needed, but it didn't get off the ground.

**Pull request merged**